src/analysis/bounded_variation.lean

```diff
@@ -207,6 +207,56 @@ begin
   simp [u, edist_comm],
 end
 
+lemma lower_continuous_aux {ι : Type*} {F : ι → α → E} {p : filter ι}
+  {f : α → E} {s : set α} (hF : tendsto_uniformly_on F f p s) :
+  ∀ (v : ℝ≥0∞), v < evariation_on f s → (∀ᶠ (n : ι) in p, v < evariation_on (F n) s) :=
```

Comment:
```suggestion
  {v : ℝ≥0∞} (hv : v < evariation_on f s) : ∀ᶠ (n : ι) in p, v < evariation_on (F n) s :=
```
Put as much context as possible left of the colon. This is more readable in general.